FreeRTOS-Plus/Demo/AWS/Fleet_Provisioning_Windows_Simulator/Fleet_Provisioning_With_CSR_Demo/DemoTasks/FleetProvisioningDemoExample.c

```diff
@@ -376,7 +399,7 @@ static bool prvSubscribeToCsrResponseTopics( void )
 
     if( xStatus == true )
     {
-        xStatus = xSubscribeToTopic( FP_CBOR_CREATE_CERT_REJECTED_TOPIC,
+        xStatus = xSubscribeToTopic( &xMqttContext, FP_CBOR_CREATE_CERT_REJECTED_TOPIC,
```

Comment:
These paramaters are probably cleaner on their own line like you have on 389. Same for later instances

Reply:
Fixed